### DISCUSSION TITLE
Approved but not getting the Benefits

### DISCUSSION THREAD

**Discussion by ishwetraj17** [community/community#135113] [GitHub Education]
### Hi everyone,

I applied two days back and was approved, but my student developer pack isn't ready. Please help me out with this. @mgriffin @zackfern @elisealix22 @martinwoodward @davidstaheli 
<img width="314" alt="Screenshot 2024-08-08 at 5 03 46 PM" src="https://github.com/user-attachments/assets/4408f643-20aa-4535-af20-10d52a4688c4">
<img width="773" alt="Screenshot 2024-08-08 at 5 07 24 PM" src="https://github.com/user-attachments/assets/591f0ed5-baf1-4c37-9414-6b47335d1020">

**Comment by davevad93** [comment]
Hi @ishwetraj17 ,

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @ishwetraj17 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
